js/index.js

```diff
@@ -36,7 +36,108 @@ const siteContent = {
     "copyright" : "Copyright Great Idea! 2018"
   },
 };
+//
+var log = document.querySelector('body');
 
+window.addEventListener('load', (event) => {
+    log.animate({
+    backgroundColor: ['black','white'],
+    opacity: [ 0, 0.9, 1 ],
+    offset: [ 0, 0.8 ], // Shorthand for [ 0, 0.8, 1 ]
+    easing: [ 'ease-in', 'ease-out' ],
+  }, 2000);
+});
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+//## Task 1: Create selectors to point your data into elements
+// * [ ] Create selectors by using any of the DOM element's methods
+// * [ ] Note that IDs have been used on all images.  Use the IDs to update src path content
+//
+let ctaImg = document.querySelector("#cta-img");
+ctaImg.src = 'img/header-img.png';
+
+window.addEventListener('load', (event)=>{
+  ctaImg.animate({
+    transformOrigin: ["80%", "40%"],
+    offset: [ 0, 0.8 ], // Shorthand for [ 0, 0.8, 1 ]
+    easing: [ 'ease-in', 'ease-out' ],
+  }, 8000);
+})
+
+// ## Task 2: Update the HTML with the JSON data
+// * [ ] Remember, NO direct updating of the HTML source is allowed.
+// * [ ] Using your selectors, update the content to match the example file.
+// * [ ] Remember to update the src attributes on images
+//
+let appendChild = document.createElement('a')
+let prependChild = document.createElement('a')
+
+var aS = document.querySelector('nav');
+aS.appendChild(appendChild);
+aS.prepend(prependChild);
```

Comment:
you appended the children, but you didn't set any textContent so it's not showing?